node/rpc/src/common_api.rs

```diff
@@ -2,17 +2,19 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use jsonrpc_v2::Error as JsonRpcError;
-use std::convert::TryInto;
 
-use fil_types::build_version::{user_version, APIVersion, Version, RUNNING_NODE_TYPE};
+use fil_types::build_version::{APIVersion, Version};
 use rpc_api::common_api::*;
+use semver::Version as SemVer;
 
-pub(crate) async fn version(block_delay: u64) -> Result<VersionResult, JsonRpcError> {
-    #[allow(clippy::needless_borrow)]
-    let v: Version = (&*RUNNING_NODE_TYPE.read().await).try_into()?;
+pub(crate) async fn version(
+    block_delay: u64,
+    forest_version: &'static str,
+) -> Result<VersionResult, JsonRpcError> {
+    let v = SemVer::parse(env!("CARGO_PKG_VERSION")).unwrap();
```

Comment:
Let's use the forest version rather than the rpc version.

Reply:
Ok but then I need to pass somehow forest semver.

Reply:
You can change the type of `forest_version` to be `semver::Version`, right?

Reply:
Or even just parse the `forest_version` string. :)

Reply:
Yes, this should work as `forest_version` is a valid semver string :). We just need to make sure it will always respect the syntax, ie. if we later use pre-release identifier, build number etc.